src/gr11review/part2/Utility.java

```diff
@@ -1,8 +1,459 @@
 package gr11review.part2;
+import java.io.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Collections;
 
-public class Utility {
 
+
+public class Utility{
+    
+    //Start of Alvan's code
     
+    /**
+    * Given a string, return the sum of the numbers appearing in the string,
+    * ignoring other characters.
+    * @param str the given string
+    * @param intArr an array that stores all the numbers extracted from the string
+    * @return the sum of the numbers appearing in the string
+    * @author Alvan Chui
+    */
+    public static int sumNumbers(String str){
+        //declare variables
+        int sum = 0;
+        //store all the numbers value in an array
+        String inputstring = str.replaceAll("[^0-9]", "@");
+        String[] intArr = inputstring.split("[@]",0);
+        //calculate the sum
+        for (String intSubString : intArr){
+            if(intSubString != "")
+                sum += Integer.parseInt(intSubString, 10);        
+        }
+        return sum;
+    }
 
+    /**
+    * Given a name of file that contains a single word on each line,
+    * returns the word that is alphabetically first.
+    * @param filenametxt the given name of file
+    * @param words an arraylist that stores every words in the file
+    * @return the word that is alphabetically first
+    * @author Alvan Chui
+    */
+    public static String alphaWord(String filenametxt) throws IOException{
+        String filepath = new File("").getAbsolutePath();
+        String fullFilePath = filepath + "/bin/gr11review/part2/" + filenametxt;
```

Comment:
looks like the files are stored in `src` not in `bin`.  since these text files are not compiled, they should be referenced at src